### ISSUE TITLE
Hint: update the search field when marker is dragged.

### ISSUE THREAD

**Issue by GMConsultant** [nthmedia/entry-gps-coordinates#13]
This code could be added to update the search field while moving the marker:

file: /src/template/_components/fields/EntryCoordinates_input.twig
line: 72

```
var latlng = new google.maps.LatLng(this.getPosition().lat(), this.getPosition().lng() );
var geocoder = geocoder = new google.maps.Geocoder();
geocoder.geocode({ 'latLng': latlng }, function (results, status) {
   if (status == google.maps.GeocoderStatus.OK) {
       if (results[1]) {
          document.getElementById("fields-location-search").value = results[1].formatted_address
       }
   }
});
```
With this improvement when the field is saved the address is the "real" one.

**Comment by niektenhoopen** [comment]
Thank you for thinking along 😀 I actually built it like that at first but there was a reason why I decided otherwise. I think it had to do with locations/addresses not precise enough or wanting to slightly adjust the marker for specific addresses. 

I have to check how/if I can implement this without affecting users that might use it differently, I don't want them to run into issues if I change the behaviour. I'll get back to you but it could be next week when I find time to research it. This week is fully packed with projects.
